cmd/fyne_demo/main.go

```diff
@@ -251,16 +251,8 @@ func makeNav(setTutorial func(tutorial tutorials.Tutorial), loadPrevious bool) f
 	return container.NewBorder(nil, themes, nil, nil, tree)
 }
 
-func shortcutFocused(s fyne.Shortcut, w fyne.Window) {
-	switch sh := s.(type) {
-	case *fyne.ShortcutCopy:
-		sh.Clipboard = w.Clipboard()
```

Comment:
From what I can tell from the code, it was always setting the clipboard twice. First when creating the struct and then this function made a type switch and set it again?

Reply:
@andydotxyz the shortcut is already setup with the clipboard at instanciation. No need to do it a second time right?

Reply:
> the shortcut is already setup with the clipboard at instanciation.

In the instantiation you make, yes - but check other usages too - like in Entry...

`e.shortcut.AddShortcut(&fyne.ShortcutCopy{}, ...)`

Reply:
I see. It may be better to get ShortcutCopy/Cut/Paste from the App though instead of relying on the user to properly instantiate it. In the meantime, I will restore the check in the fyne_demo app.

Reply:
> It may be better to get ShortcutCopy/Cut/Paste from the App though instead of relying on the user to properly instantiate it

I agree with this statement, however having the fields be `nil` would be a breaking change, so thanks for re-instating it.